playground/backend/cmd/server/controller.go

```diff
@@ -74,6 +74,9 @@ func (controller *playgroundController) RunCode(ctx context.Context, info *pb.Ru
 	if err = utils.SetToCache(ctx, controller.cacheService, pipelineId, cache.LogsIndex, 0); err != nil {
 		return nil, errors.InternalError("Run code()", "Error during set value to cache: %s", err.Error())
 	}
+	if err = utils.SetToCache(ctx, controller.cacheService, pipelineId, cache.Canceled, false); err != nil {
+		return nil, errors.InternalError("Run code()", "Error during set cancel flag to cache as a false")
```

Comment:
add `code_processing.DeleteFolders(pipelineId, lc)`

Reply:
@AydarZaynutdinov probably not necessary if the cancel is false

Reply:
Actually is we need to set false before the processing starts. In another case in the method where we try to get this value, we will receive an error:
https://github.com/apache/beam/pull/16278/files/f389c3c2f4a79b46e3bc7ec1d0c4ea8ac1d6a1bb#diff-4be52118bb3b1a645c83e5f5d93d5489fe351cc59e234323240625f61a17fb4cR337-R338

Reply:
Done.